apps/server/test/lexical/server/code_intelligence/completion/translations/function_test.exs

```diff
@@ -0,0 +1,97 @@
+defmodule Lexical.Server.CodeIntelligence.Completion.Translations.FunctionTest do
+  use Lexical.Test.Server.CompletionCase
+
+  describe "function completions" do
+    test "deprecated functions are marked", %{project: project} do
+      assert {:ok, completion} =
+               project
+               |> complete("Enum.filter_map|")
+               |> fetch_completion("filter_map")
+
+      assert completion.detail
```

Comment:
Are you asserting the `completion.detail`  not nil?

Reply:
correct; that there's something there.